week-2-practise/level-1/2-validation.js

```diff
@@ -2,7 +2,7 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
+var isBigEnough = true; // ONLY EDIT THIS LINE
```

Comment:
Hi, the expression should be num >= 10 rather than true so that it would still work even if num wasn't 10